oak/server/rust/oak_runtime/src/config.rs

```diff
@@ -16,21 +16,36 @@
 
 //! Functionality covering configuration of a Runtime instance.
 
-use crate::{RuntimeConfiguration, RuntimeProxy};
+use crate::{runtime::RuntimeProxy, Runtime, RuntimeConfiguration};
+use log::{error, info};
 use oak_abi::OakStatus;
+use std::sync::Arc;
 use tonic::transport::Certificate;
 
-/// Configures a [`RuntimeProxy`] from the given [`RuntimeConfiguration`] and begins execution.
+/// Configures a [`Runtime`] from the given [`RuntimeConfiguration`] and begins execution.
 ///
 /// Returns a [`RuntimeProxy`] for an initial implicit Node, and a writeable [`oak_abi::Handle`] to
 /// send messages into the Runtime. Creating a new channel and passing the write [`oak_abi::Handle`]
 /// into the runtime will enable messages to be read back out from the [`RuntimeProxy`].
-pub fn configure_and_run(
-    config: RuntimeConfiguration,
-) -> Result<(RuntimeProxy, oak_abi::Handle), OakStatus> {
+pub fn configure_and_run(mut config: RuntimeConfiguration) -> Result<Arc<Runtime>, OakStatus> {
     let proxy = RuntimeProxy::create_runtime(&config.app_config, &config.grpc_config);
+    let config_map = config.config_map.take();
     let handle = proxy.start_runtime(config)?;
-    Ok((proxy, handle))
+
+    // Pass in the config map over the initial channel.  For consistency, ensure
+    // that exactly one message is sent over the initial channel whether there's
+    // a config map provided or not.
+    let sender = crate::io::Sender::new(handle);
+    info!("Send in initial config map");
+    sender.send(config_map.unwrap_or_default(), &proxy)?;
```

Comment:
What will happen if a module doesn't read ConfigMap?
Will it be an error of unread message, or it's a normal behavior?

Reply:
No error, it's normal behaviour (e.g. every for example except the`trusted_information_retrieval` one) &ndash; 
 if the Node doesn't read the message nor close the channel that holds the message, then it will just sit there unread.